force-app/main/default/lwc/edaSettingsNavigation/edaSettingsNavigation.js

```diff
@@ -82,8 +88,12 @@ export default class EdaSettingsNavigation extends LightningElement {
         },
         programSettings: {
             programSettings: stgProgramsSettingsNav,
-            programAutoEnrollmentMappings: "Program Auto-Enrollment Mappings",
+            programAutoEnrollmentMappings: stgAutoEnrollmentProgramTitle,
             programEnrollmentDeletions: "Program Enrollment Deletions",
```

Comment:
This is still hardcoded. Can you tend to this as well? We should have a label for this.